apps/redi-connect/src/components/organisms/ConfirmMentorship.tsx

```diff
@@ -73,7 +80,32 @@ const ConfirmMentorship = ({ match }: ConfirmMentorshipProps) => {
 
   return (
     <>
-      <Button onClick={() => setModalActive(true)}>Accept</Button>
+      {hasReachedDesiredMenteeLimit ? (
+        <>
+          {hasReachedDesiredMenteeLimit &&
```

Comment:
You are already checking if `hasReachedDesiredMenteeLimit`is true two lines above, so you can safely remove this check here, and only check the `menteeCountCapacity`.

Reply:
You are right, thank you!